src/Pages/Login/Login.js

```diff
@@ -1,31 +1,66 @@
-import React from "react";
+import React, { useContext } from "react";
 import "./Login.css";
 import { Link } from "react-router-dom";
 
+import { useNavigate } from "react-router-dom";
+import { PrivateRouteContext } from "../../Context/privateRoute-context";
+import { AuthContext } from "../../Context/auth-context";
+
 export default function Login() {
+  const navigate = useNavigate();
+  const { userName, setUserName, password, setPassword, setLogged } =
+    useContext(AuthContext);
+  const { privateRoute, setPrivateRoute } = useContext(PrivateRouteContext);
+
+  const handleLogin = (e) => {
+    e.preventDefault();
+
+    if (userName === "adarshbalika@neog.camp" && password === "password") {
+      setPrivateRoute(true);
```

Comment:
This logic will not work for other users. you can check the local storage for auth token. If it is there set the route to private.  or set this private route during login/signup itself.

Reply:
I have implemented only basic authentication for now. I will implement authentication with tokens later.